pkg/ensurance/cm/policy.go

```diff
@@ -0,0 +1,19 @@
+package cm
+
+import (
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/kubernetes/pkg/kubelet/cm/cpumanager/state"
+)
+
+type policyName string
+
+// Policy implements logic for pod container to CPU assignment.
```

Comment:
Please add more comment info for interface

Reply:
ok